src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,22 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.List;
+
+/**
+ * Sort addressbook alphabetically
```

Comment:
Perhaps we can be more precise with this, will the sorting be for name? If it is sorted by name is it in lexicographical order? =)